msteams-platform/concepts/build-and-test/shared-channels.md

```diff
@@ -59,6 +62,26 @@ You can collaborate with external members outside of your organization using sha
 
 ## Get shared channel membership
 
+Use the [List allMembers API](/graph/api/channel-list-allmembers?view=graph-rest-beta&tabs=http) to retrieve all members (**direct** and **indirect**) of a shared channel.
+
+### Identify Direct vs. Indirect Members
+
+You can identify whether a member of a shared channel is direct or indirect by checking the **@microsoft.graph.originalSourceMembershipUrl** annotation. This property identifies the source of a member’s access to a shared channel, as shown in the following table.
+
+|Member Type |Annotation Present?  |Description  |
+|---------|---------|---------|
+|Direct Member  |  **No**      |  The user is added directly to the shared channel.  |
+|Indirect Member|  **Yes**     | The user accesses the shared channel through another team. The annotation includes a URL that points to the source team.     |
+
+> [!NOTE]
+> You might receive duplicate notifications when a member is added to a shared channel. This scenario can happen if the member is already part of the shared channel directly or through another linked team.
+>
+> To avoid duplicate notifications:
```

Comment:
I don't think it can be avoided. But More like identifying if this is a duplicate or not and take appropriate action.